docs/advanced/addons/lvm-local-storage.md

```diff
@@ -0,0 +1,153 @@
+
+1. Create a new StorageClass and choose the Provisioner as **LVM**.
+
+![](/img/v1.4/csi-driver-lvm/create-lvm-sc-01.png)
+
+2. Choose the target node. The LVM is the local volume, so we need to choose the target node for the future workload.
+
+![](/img/v1.4/csi-driver-lvm/create-lvm-sc-02.png)
+
+3. Choose the target VG name. The VG name is the volume group name you created before.
+
+![](/img/v1.4/csi-driver-lvm/create-lvm-sc-03.png)
+
+4. Choose the target LV type. Currently we support **striped** and **dm-thin**.
+
+![](/img/v1.4/csi-driver-lvm/create-lvm-sc-04.png)
+
+5. Click **Save**. Then you can see the new StorageClass on the **Storage** page.
+
+![](/img/v1.4/csi-driver-lvm/create-lvm-sc-05.png)
+
+## Create Volume with LVM
+
+You can create a new volume and attach it to the workload(VM). These steps are shown below:
+
+1. Create a new volume with the new StorageClass you created before.
+
+:::note
+
+The volume status `Not Ready` is normal because the LVM volume will be created when the first workload is created.
+
+:::
+
+![](/img/v1.4/csi-driver-lvm/create-lvm-volume-01.png)
+
+2. Attach the volume to the workload(VM).
+
+:::note
+
+The LVM volume is a local volume, so you need to ensure the SC target node is the same as the VM scheduled node.
+
+:::
+
+![](/img/v1.4/csi-driver-lvm/attach-lvm-volume-01.png)
+
+3. After the volume attached, you can see the volume status is `Ready`.
+
+![](/img/v1.4/csi-driver-lvm/attach-lvm-volume-02.png)
+
+You can also create a new workload(VM) with the volume of the StorageClass you created before. The VM will be scheduled to the target node with local storage for the volume.
+
+![](/img/v1.4/csi-driver-lvm/create-vm-with-lvm-volume-01.png)
+
+![](/img/v1.4/csi-driver-lvm/create-vm-with-lvm-volume-02.png)
+
+## Config Snapshot/Backup for LVM volume
+
+The LVM volume supports the snapshot and backup feature but you need to update the **csi-driver-config** in **Settings**. The steps are shown below:
+
+1. Go to the **Settings** page and click the **Edit Setting** button on **csi-driver-config**.
+
+![](/img/v1.4/csi-driver-lvm/update-csi-driver-config-01.png)
+
+2. Add new **Provisioner**, **Volume Snapshot Class Name** and **Backup Volume Snapshot Class Name**.
+
+![](/img/v1.4/csi-driver-lvm/update-csi-driver-config-02.png)
+
+3. Now, you can create a new snapshot or backup for the LVM volume. (Here, snapshot is an example)
+
+![](/img/v1.4/csi-driver-lvm/vm-take-snapshot-with-lvm-01.png)
+
+4. check the snapshot status
+
+![](/img/v1.4/csi-driver-lvm/vm-take-snapshot-with-lvm-02.png)
+
+## Support features for LVM volume
+
+:::caution
+
+Although the backup feature looks good, it is not verified and supported.
+
+:::
+
+- **Snapshot**: Create a snapshot for the LVM volume.
+- **Resize**: Resize the LVM volume.
+- **Clone**: Clone the LVM volume.
```

Comment:
Why do we include backup creation in the list of supported features if it's not supported? Also, how are "verified" and "supported" different?

cc: @bk201 @asettle

Reply:
We did not block the backup button in v1.4.0. And it is too late to add this change.

I discussed with Kiefer that it might not work because of the backup mechanism.
So, we need to add this warning to the document.

The verified and supported will be the same thing. We need to verify, and then we can say it is support.

Reply:
I suggest the term "not supported" to make it very clear.